docs/configuration/http_conn_man/route_config/route.rst

```diff
@@ -300,3 +304,24 @@ runtime_key_prefix
 
   **Note:** If the sum of runtime weights exceed 100, the traffic splitting behavior
   is undefined (although the request will be routed to one of the clusters).
+
+.. _config_http_conn_man_route_table_opaque_config:
+
+Opaque Config
+-------------
+
+Additional configuration can be provided to filters through the "Opaque Config" mechanism.  A
+list of Key/Value pairs are specified in the route config.  The configuration is uninterpreted
+by envoy and can be accessed within a user-defined filter.
+
+.. code-block:: json
+
+  [
+    {"name": "...", "value": "..."}
```

Comment:
I thought Matt's comment was about how the JSON is returned. An unorderd map would be a flat JSON Object where all values are strings. That's a bit different from an array of JSON objects with name and value pairs. This format seems like it's the same structure represented in a more obtuse way.